src/Hng.Infrastructure/Services/OrganizationInviteService.cs

```diff
@@ -0,0 +1,62 @@
+using System.Security.Cryptography;
+using System.Text;
+using Hng.Domain.Entities;
+using Hng.Infrastructure.Repository.Interface;
+using Hng.Infrastructure.Services.Interfaces;
+
+namespace Hng.Infrastructure.Services;
+
+public class OrganizationInviteService(IRepository<Organization> Organizationrepository, IOrganizationInviteRepository repository) : IOrganizationInviteService
+{
```

Comment:
Remove everything concerning `IOrganizationInviteRepository`. Use `IRepository<OrganisationInvite`, that will give you all the methods to access your table.